README.md

```diff
@@ -56,15 +56,15 @@ Yoti also allows you to enable user details verification from your mobile app by
 To download and install the Yoti SDK and its dependencies, simply run the following command from your terminal:
 
 ```Go
-go get "github.com/getyoti/yoti-go-sdk"
+go get "github.com/getyoti/yoti-go-sdk/v2"
```

Comment:
Consider mentioning modules are now used, so it's not necessary get a copy or fetch all dependencies as the Go toolchain will fetch them as necessary. Unless someone wishes to run the examples or make a contribution it's sufficient to just start using it/add a `require` to `go.mod` and the toolchain will take care of the rest.